src/redux/books.js

```diff
@@ -0,0 +1,52 @@
+const ADD_BOOK = 'react-bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'react-bookstore/books/REMOVE_BOOK';
+
+const initialState = {
+  books: [
+    {
+      id: 1,
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+      category: 'Action',
+    },
+    {
+      id: 2,
+      title: 'Dune',
+      author: 'Frank Herbert',
+      category: 'Science Fiction',
+    },
+    {
+      id: 3,
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+      category: 'Economy',
+    },
```

Comment:
- [ ] I will suggest you follow the project requirement kindly go through the requirement according to the project requirement and consider to Define a slice of state for books that:
Includes an array of books (initial state: empty array)
Includes a reducer that adds a book
Includes a reducer that removes a book